review/passenger-service/src/main/java/cz/muni/fi/airportmanager/passengerservice/resource/PassengerResource.java

```diff
@@ -120,6 +121,52 @@ public Uni<RestResponse<Passenger>> get(@Parameter(name = "id", required = true,
                 });
     }
 
+    /**
+     * Get passengers for a flight by flight id
+     * @param flightId flight id
+     *                 @return list of passengers for a flight
+     */
+    @GET
+    @Path("/flight/{flightId}")
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(summary = "Get passengers for a flight by flight id")
+    @APIResponse(
+            responseCode = "200",
+            description = "List of passengers for a flight",
+            content = @Content(
+                    mediaType = APPLICATION_JSON,
+                    schema = @Schema(implementation = Passenger.class, required = true),
+                    examples = @ExampleObject(name = "flight", value = Examples.VALID_PASSENGER_LIST)
+            )
+    )
+    public Uni<RestResponse<List<Passenger>>> getPassengersForFlight(@Parameter(name = "flightId", required = true, description = "Flight id") @PathParam("flightId") Long flightId) {
```

Comment:
@grygy, isn't this already done in lecture 04? This PR should be only code changed in 05.

Reply:
No, this is added for testing lecture